docs/index.html

```diff
@@ -691,8 +691,15 @@
                   <a href="https://github.com/quintilesims/layer0/edit/master/docs/index.md" title="Edit this page" class="md-icon md-content__icon">&#xE3C9;</a>
                 
                 
-                <p><img alt="Layer0 Logo" src="./static/logo_rs.png" /></p>
+                <!-- ![Layer0 Logo](/static/logo_rs.png) -->
```

Comment:
you shouldn't change anything in the `docs` directory unless doing a release. You should un-commit these changes.

Reply:
Yup. I'll remove this.